V/vegafusion/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+using BinaryBuilder
+
+name = "vegafusion"
+version = v"0.0.4"
+
+sources = [
+    GitSource("https://github.com/vegafusion/vegafusion.git",
+                  "c0927268fc7544878c58a606e01503ba3fed615e"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+export TARGET_CC=$CC
+
+cd ${WORKSPACE}/srcdir/vegafusion/vegafusion-server/
+cargo build --release
+mkdir -p "${bindir}"
+cp "../target/${rust_target}/release/vegafusion-server${exeext}" "${bindir}/."
```

Comment:
If you want you can simplify with
```suggestion
install -Dm 755 "../target/${rust_target}/release/vegafusion-server${exeext}" "${bindir}/vegafusion-server${exeext}"
```

Reply:
I made a number of changes at once, but now I get 

```bash
curl: Can't open './products/vegafusion*x86_64-unknown-freebsd*.tar.gz'!
```

Assume this is related to this fix, but I'll roll back and check. :)

Reply:
> Assume this is related to this fix, but I'll roll back and check. :)

Not at all (the failure is even after the tarball is created, how it can be related?), it's due to the fact that the directory name _**must**_ match the `name` of the package, so you have to rename the directory too